hack/update-bazel.sh

```diff
@@ -30,10 +28,11 @@ rm -f "${KUBE_ROOT}/pkg/generated/openapi/zz_generated.openapi.go"
 export GOBIN="${KUBE_OUTPUT_BINPATH}"
 PATH="${GOBIN}:${PATH}"
 
-# Install tools we need, but only from vendor/...
-go install ./vendor/github.com/bazelbuild/bazel-gazelle/cmd/gazelle
-
-go install ./vendor/github.com/kubernetes/repo-infra/kazel
+pushd "${KUBE_ROOT}" > /dev/null 2>&1
+  # Install tools we need, but only from vendor/...
+  go install ./vendor/github.com/bazelbuild/bazel-gazelle/cmd/gazelle
```

Comment:
we could make these `go install k8s.io/kubernetes/vendor/...` but I'm not sure that's better.